doc/api/zlib.md

```diff
@@ -39,11 +65,41 @@ pipeline(source, gzip, destination, (err) => {
     process.exitCode = 1;
   }
 });
+```
 
-// Or, Promisified
+Or, using promisified `pipeline`:
```

Comment:
I think "promosified" usually refers to an API that went through `util.promisify`, it feels weird to use it to refer to a native promise API.
```suggestion
Or, using the promise `pipeline` API:
```